src/main/backend.ts

```diff
@@ -150,30 +142,95 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
     const hubScope = getCurrentHub().getScope();
     if (hubScope) {
       hubScope.addScopeListener(updatedScope => {
-        const cloned = Scope.clone(updatedScope);
+        const scope = Scope.clone(updatedScope);
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-        (cloned as any)._eventProcessors = [];
+        (scope as any)._eventProcessors = [];
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-        (cloned as any)._scopeListeners = [];
+        (scope as any)._scopeListeners = [];
+
         // if we use the crashpad minidump uploader we have to set extra whenever the scope updates
         if (this._options.useCrashpadMinidumpUploader === true) {
-          getCurrentHub().captureEvent(
-            {
-              // @ts-ignore __INTERNAL_MINIDUMP is not assignable to event
-              __INTERNAL_MINIDUMP: true,
-            },
-            {
-              data: {
-                __sentry__: true,
-              },
-            },
+          forget(
+            this._getNativeUploaderEvent(scope).then(event => {
+              // Update the extra parameters in the main process
+              const mainParams = this._getNativeUploaderExtraParams(event);
+              for (const key of Object.keys(mainParams)) {
+                crashReporter.addExtraParameter(key, mainParams[key]);
+              }
+
+              // Modify the event to point to renderer crash
+              event.extra = { ...event.extra, crashed_process: 'renderer' };
```

Comment:
_nit_: Might be faster to just overwrite the one field, assuming `extra` is always set:

```suggestion
              event.extra.crashed_process = 'renderer';
```

Reply:
`extra` does appear to always be set but it's optional in the type definitions which means TypeScript throws an error with  this:
```ts
event.extra.crashed_process = 'renderer';
```

Reply:
oh, this code is going anyway